config/routes.rb

```diff
@@ -412,6 +412,9 @@
   get "/:username/:slug" => "stories#show"
   get "/:username" => "stories#index"
 
+  # ea - the fetch call in the youtube partial should come here
+  get "/:username/videos/youtube" => "youtube#index"
```

Comment:
I am still a little confused by this route. The route that we agreed would render the videos is get "/:username/:slug" => "stories#show". I trust your work but will have to have you explain this one to me.

Reply:
No problem. This route is called from the partial we are rendering for the videos in thee articles/show view. It connects to the youtube controller and serializes the data for the youtube videos (from a facade which initializes a service).